### DISCUSSION TITLE
Question about phreatic surface and hydraulic boundary conditions

### DISCUSSION THREAD

**Discussion by Tu-Shiqin** [Anura3D/Anura3D_OpenSource#83] [Q&A] [answered]
Hi dear all,
I'm simulating the failure process of submerged tunnel by Anura3D, but some problems about phreatic surface have been exposed when the results with different initial conditions are compared in Paraview.  Fig. 1 is the table of parameters, which the saturated soil is selected as the stratum.
![d2ba0fd886eddb7a9288b96f736f944](https://user-images.githubusercontent.com/125841317/226180379-fa0f1176-b21a-4b9f-9fe1-538bd0c5d50a.png)
After the soil belong to the tunnel was excavated, the stratum in front of tunnel face collapse and material points flooded into the tunnel, as shown in Fig. 2 (real time is 3s).
![image](https://user-images.githubusercontent.com/125841317/226180908-b4568df6-498f-4438-b5f4-31ea310a90f7.png)
And in the second simulation test, I defined a phreatic surface above the soil interface as shown in Fig. 3.
![image](https://user-images.githubusercontent.com/125841317/226181012-396b0dd9-d2e3-46ed-98ed-dc60b5b73407.png)
The initial stress field and liquid pressure field are shown in Fig. 4 and Fig. 5, which are as expected. 
![920a7f7b1ba4314585b5728dff6ab7e](https://user-images.githubusercontent.com/125841317/226181216-9ef9c069-66b8-4bfd-ade2-c178e9547cc5.png)
![d29749d0dfdb3463f61d3ec62685b73](https://user-images.githubusercontent.com/125841317/226181217-9e857710-deb0-4348-9e12-90ea4481a5ba.png)
But after excavation, The stratum is stabled without any supporting applied to the tunnel face, as shown in Fig. 6 (real time=3s).
As a rule of thumb ,the increase of water head has an adverse effect on tunnels as well as slope etc. I want to know how the phreatic surface is considered in the every calculation time step of Anura3D and what should I do to use phreatic surface and hydraulic boundary conditions correctly to get accurate results in this simulation. I am appreciate you for any help! Hope for your comments.

Best 
Tu

**Comment by Tu-Shiqin** [comment]
Fig. 6
![e3addb3cfd6bf6a858c398677b39cb1](https://user-images.githubusercontent.com/125841317/226181934-b18401ae-78a0-4082-bf78-0aa53c72f9b2.png)

**Reply by FrancescaCeccato** [reply, marked as answer]
Can you plot the pore pressure? I have the feeling that if the material has a low permeability, at short term it will be stable because of the suction generated by the water.
The phreatic surface is only used to determine the initial pore pressure at the first load step, it is an INITIAL CONDITION.
